lib/genesis/helpers.go

```diff
@@ -212,7 +213,7 @@ func buildRawArrayInterface(a []interface{}, kv *keyValue) {
 			kv.iVal = append(kv.iVal, tba)
 		case float64:
 			// TODO: determine how to handle this error
-			encVal, _ := scale.Encode(uint64(v2))
+			encVal, _ := scale.Marshal(uint64(v2))
```

Comment:
really should handle this error

Reply:
@timwu20 I handled it with a panic, but can refactor to return an error if that is prefered

Reply:
That is preferred.  We should never panic in the codebase.